src/Tribe/Editor/Blocks/Tickets.php

```diff
@@ -190,18 +190,28 @@ public function ticket_availability( $tickets = array() ) {
 
 		// Parse the tickets and create the array for the response
 		foreach ( $tickets as $ticket_id ) {
-			$ticket    = Tribe__Tickets__Tickets::load_ticket_object( $ticket_id );
+			$ticket = Tribe__Tickets__Tickets::load_ticket_object( $ticket_id );
 
-			if ( empty( $ticket ) ) {
+			if (
+				! $ticket instanceof Tribe__Tickets__Ticket_Object
+				|| empty( $ticket->ID )
+			) {
 				continue;
 			}
 
-			$available = $ticket->available();
+			/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
+			$tickets_handler = tribe( 'tickets.handler' );
+
+			$available = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
+
 			$response['tickets'][ $ticket_id ]['available'] = $available;
 
+			/** @var Tribe__Tickets__Editor__Template $tickets_editor */
+			$tickets_editor = tribe( 'tickets.editor.template' );
```

Comment:
This var set should go above and outside the `foreach()`

Reply:
good catch, again. done.